linker/Mono.Linker.Steps/ResolveFromAssemblyStep.cs

```diff
@@ -67,7 +67,7 @@ static void SetAction (LinkContext context, AssemblyDefinition assembly, Assembl
 		{
 			TryReadSymbols (context, assembly);
 
-			context.Annotations.SetAction (assembly, action);
+			context.SetAction (assembly, action);
 		}
```

Comment:
@vargaz I don't think this change is correct. Consider a case where you have -a SomeCoreAssembly -c link. With your change the action for SomeAssembly will be set to AssemblyAction.Link instead of AssemblyAction.Copy. What was the reason for changing this line?